### PR TITLE
Code Climate updates

### PR DESCRIPTION
**Pull request by joesiewert**
Trying to sync up with changes on https://docs.codeclimate.com/docs/codeship-ci-test-coverage-example

### CODE REVIEW

File: _general/integrations/codeclimate.md [hunk 1]
```diff
@@ -166,22 +166,21 @@ At the end of each [parallel pipeline]({{ site.baseurl }}{% link _basic/builds-a
...
+- As a command, run via the [custom-script deployment option]({{ site.baseurl }}{% link _basic/continuous-deployment/deployment-with-custom-scripts.md %}). This means code coverage for parallel testing will only run on branches you have configured deployments.
 
 - As an additional test step placed at the end of one of your parallel test pipelines. This method will require additional logic to be written to pause the script while it queries your external storage service for the existence of the appropriately-named coverage reports for all the additional pipelines, so that it doesn't erroneously combine coverage reports for pipelines that are still in progress.
 
 The code to use to end the parallel coverage report is:
 
 ```shell
-cc-test-reporter sum-coverage --output - --parts $PARTS coverage/codeclimate.*.json | \
-    ./cc-test-reporter upload-coverage
+cc-test-reporter sum-coverage --output - --parts $PARTS coverage/codeclimate.*.json | \ ./cc-test-reporter upload-coverage
```
Comment (mlocher) [inline _general/integrations/codeclimate.md]: If this is all on a single line now, the `\` is superfluous

File: _general/integrations/codeclimate.md [hunk 2]
```diff
@@ -115,10 +115,10 @@ Next, at the end of your build itself, as a new test command placed after your n
 Inside the `codeclimate-assemble.sh` file, you will have:
 
 ```shell
-cc-test-reporter sum-coverage --output - --parts $PARTS coverage/codeclimate.*.json | \
+cc-test-reporter sum-coverage --output - --parts $PARTS coverage/codeclimate.*.json | \ ./cc-test-reporter upload-coverage
```
Comment (mlocher) [inline _general/integrations/codeclimate.md]: If this is all on a single line now, the `\` is superfluous

File: _general/integrations/codeclimate.md (hunk 1 above)
Comment (joesiewert) [reply _general/integrations/codeclimate.md]: Thanks, too much copy/paste ✂️

**Comment by mlocher** [comment]
Nice work, thanks 🙇

**Pull request merged**